src/functions/warnOnce.js

```diff
@@ -0,0 +1,19 @@
+'use strict';
+
+var bind = require('lodash/function/bind');
+
+var warn = window ?
+  window.console && bind(window.console.warn, console) :
+  bind(console.warn, console); // eslint-disable-line no-console
+
+var warnOnce = (function(w) {
+  var previousMessages = [];
```

Comment:
Yes, it will grow by one for each new message generated. Do you think it might a problem? Maybe we should deactivate that in the minified version?